tools/raml2html/composer.json

```diff
@@ -20,7 +20,9 @@
         "raml-org/raml-php-parser": "dev-master",
         "twig/twig": "^2.0",
         "symfony/filesystem": "^4.2",
-        "ramsey/uuid": "^3.9"
+        "ramsey/uuid": "^3.9",
+        "twig/markdown-extra": "^3.5",
+        "league/commonmark": "^2.3"
```

Comment:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires league/commonmark ^2.3 -> satisfiable by league/commonmark[2.3.0, ..., 2.3.8].
    - league/commonmark[2.3.0, ..., 2.3.8] require php ^7.4 || ^8.0 -> your php version (7.2.34) does not satisfy that requirement.
```

So, this addition is not compatible with `"php": "^7.2",` above.

Reply:
Quick fix in c34be1469801e0ce0035448577074fbbe991a4fa for PHP 7.4 w/ few warnings at runtime.

A better fix for PHP 8.1+ will come in #1871

Reply:
@adriendupuis we will need to take the raml parser and everything else related to it under our organization, because that library is in practice abandoned.